docs/pages/comment_parsing.rst

```diff
@@ -126,3 +131,75 @@ The full program
         if comment.body == "Hello" and comment.id not in already_done:
             comment.reply(' world!')
             already_done.add(comment.id)
+
+Navigating Trees
+------------------
+
+Comments are in trees, each submission has a comment forest. However with
+some simple manipulation, we can manipulate
+
+Special Comment Methods
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Going up and down comment trees is useful for various scripts and is sometimes
+needed, for replying to the parent or ancestral submitter, however natively
+there aren't any easy solutions. Due to this PRAW has implemented ways to
+do this from Comment objects.
+
+Comment objects have the following special methods:
+
+* ``get_tree()``
```

Comment:
Do we specifically need the `get_` prefixes on these methods? I realize there are a fair number of methods in PRAW with `get_` prefixes already, but given enough time I'd like to deprecate them as `get` is implied when only passing sufficient arguments to obtain data, and `set` is implied when passing additional arguments.